src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
```suggestion
public class ScvReportServiceImpl implements ReportService {
```
Result is also in scv format